test/spec/default_spec.rb

```diff
@@ -0,0 +1,80 @@
+require_relative "./spec_helper"
+
+describe "rubygems::default" do
+  let(:gemrc) { "#{File.join(Dir.home, ".gemrc")}" }
+  let(:global_gemrc) { Gem::ConfigFile::SYSTEM_WIDE_CONFIG_FILE }
+
+  context "with default attributes" do
+    let(:chef_run) do
```

Comment:
Since there is only one or two specs, and the nature of the specs don't have anything platform dependent to warrant looping on different platforms/versions, caching doesn't seem to make any relevant difference to the speed of the tests.